<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEvent</class>
 <widget class="QDialog" name="AddEvent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>New event</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelDateTime">
       <property name="text">
        <string>Date:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelNotifications">
       <property name="text">
        <string>Event frequency:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTimeEdit" name="timeEdit"/>
     </item>
     <item row="1" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="selectedDate">
        <date>
         <year>2021</year>
         <month>6</month>
         <day>4</day>
        </date>
       </property>
       <property name="minimumDate">
        <date>
         <year>2021</year>
         <month>6</month>
         <day>4</day>
        </date>
       </property>
       <property name="firstDayOfWeek">
        <enum>Qt::Monday</enum>
       </property>
       <property name="gridVisible">
        <bool>true</bool>
       </property>
       <property name="horizontalHeaderFormat">
        <enum>QCalendarWidget::ShortDayNames</enum>
       </property>
       <property name="verticalHeaderFormat">
        <enum>QCalendarWidget::NoVerticalHeader</enum>
       </property>
       <property name="navigationBarVisible">
        <bool>true</bool>
       </property>
       <property name="dateEditEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelNotes">
       <property name="text">
        <string>Event notes:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelEventHeader">
       <property name="text">
        <string>Event header:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="addEventButton">
       <property name="text">
        <string>Add event</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPlainTextEdit" name="plainTextEdit"/>
     </item>
     <item row="8" column="1">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Event mode:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboNotifications">
       <item>
        <property name="text">
         <string>One time</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Every hour</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Every day</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Every week</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Every month</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Time:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEventHeader"/>
     </item>
     <item row="6" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="comboMode">
       <item>
        <property name="text">
         <string>Push</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dialog</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Spam</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
